lc0/src/engine.cc

```diff
@@ -63,6 +65,10 @@ void EngineController::PopulateOptions(OptionsParser* options) {
       backends.empty() ? "<none>" : backends[0];
   options->Add<StringOption>(kNnBackendOptionsStr, "backend-opts");
   options->Add<FloatOption>(kSlowMoverStr, 0.0, 100.0, "slowmover") = 1.5;
+  options->Add<IntOption>(kMoveOverheadStr, 0, 10000, "move-overhead") = 30;
+  options->Add<IntOption>(kMinThinkingTimeStr, 0, 10000, "min-thinking-time") =
+      20;
```

Comment:
I think that unlike stockfish, for lc0 just returning policy head without thinking is better than running out of time.
Is there a reason to have default 20? Maybe swap std::min and std::max inline 111.

Reply:
No reason, I think 0 is a good default.

Reply:
I have it at 0 with my leela bot on lichess and it can make nonsense moves (with temperature on) or weak moves (fixed with some visits) much more often that with it above 0. Having a small value might be beneficial here, maybe even single digits.